.editorconfig

```diff
@@ -37,7 +37,7 @@ charset = utf-8
 indent_size = 4
 # isort plugin configuration
 known_first_party = invenio_admin
-known_third_party = invenio_db
+known_third_party = invenio_access invenio_db
```

Comment:
try comma-separated: ``invenio_access,invenio_db``
 like in: https://github.com/inveniosoftware/invenio-pidstore/blob/master/.editorconfig#L38